rds-instance-full/variables.tf

```diff
@@ -77,3 +77,8 @@ variable "ingress_rule" {
   description = "Ingress rule to open the ports towards VPC"
   default     = "postgresql-tcp"
 }
+
+variable "parameters" {
+  description = "List of parameters"
```

Comment:
While we wait for @antonbabenko - "a list of parameters for what?" Please add a more descriptive description.

Reply:
Those are parameters in parameter group of  RDS, I am going to add longer description, thanks